doc/guides/feature_compat.md

```diff
@@ -0,0 +1,95 @@
+---
+module: [kind=guide] feature_compat
+---
+
+# Lua 5.2/5.3 features in CC: Tweaked
+CC: Tweaked is based off of the Cobalt Lua runtime, which uses Lua 5.1. However, Cobalt and CC:T implement additional features from Lua 5.2 and 5.3 (as well as some deprecated 5.0 features) that are not available in base 5.1. This page lists all of the compatibility for these newer versions.
+
+## Lua 5.2
+| Feature | Supported? | Notes |
+|---------|------------|-------|
+| `goto`/labels | :x: |  |
+| `_ENV` | :large_orange_diamond: | The `_ENV` global points to `getfenv()`, but it cannot be set. |
+| `\z` escape | :heavy_check_mark: |  |
+| `\xNN` escape | :heavy_check_mark: |  |
+| hex literal fractional/exponent parts | :heavy_check_mark: |  |
+| empty statements | :x: |  |
+| `__len` metamethod | :heavy_check_mark: |  |
+| `__ipairs` metamethod | :x: |  |
+| `__pairs` metamethod | :heavy_check_mark: |  |
+| `bit32` library | :heavy_check_mark: | Replaces `bit` library, which is still available for compatibility. |
```

Comment:
I don't think it's worth mentioning the `bit` library - that's not actually documented anywhere.

Reply:
Huh, I could've sworn it was there before, but just checked and it indeed isn't (since it's not an API, it's preloaded in the BIOS).